plugins/talk-plugin-auth/client/index.js

```diff
@@ -13,6 +14,7 @@ export default {
   slots: {
     stream: [UserBox, SignInButton, SetUsernameDialog],
     login: [Login],
+    profileHeader: [Profile],
     profileHeader: [ChangeUsername],
```

Comment:
This seems to overwrite the previous entry other?

Reply:
It seems to be a merge conflict. Checking!

Reply:
Yes, this is correct. 

```
    stream: [UserBox, SignInButton, SetUsernameDialog],
    login: [Login],
    profileHeader: [Profile],
    profileSettings: [ChangePassword],
```

This PR handles the change of the email too within Profile Component (which handles both username and email changes)